tests/integration/kvm.py

```diff
@@ -176,112 +178,102 @@ def _adjust_kvm(self):
-        cmd_kvm_adj.append("guestfish -a /tmp/{image_name}.snapshot.img -i chown 0 0 /root/.ssh".format(
-            image_name=image_name))
-        cmd_kvm_adj.append("guestfish -a /tmp/{image_name}.snapshot.img -i chown 0 0 /root/.ssh/test_authorized_keys".format(
-            image_name=image_name))
-        cmd_kvm_adj.append("guestfish -a /tmp/{image_name}.snapshot.img -i chmod 0700 /root/.ssh".format(
-            image_name=image_name))
-        cmd_kvm_adj.append("guestfish -a /tmp/{image_name}.snapshot.img -i chmod 0600 /root/.ssh/test_authorized_keys".format(
-            image_name=image_name))
-        cmd_kvm_adj.append("guestfish -a /tmp/{image_name}.snapshot.img -i write-append /etc/hosts.allow 'ALL: 10.\n'".format(
-            image_name=image_name))
-        # Copy custom SSHD config for executing remote integration tests
-        # without changing the production sshd_config. This SSHD runs on
-        # port tcp/2222
-        cmd_kvm_adj.append("virt-copy-in -a /tmp/{image_name}.snapshot.img {sshd_systemd_src_file} {systemd_dst_path}".format(
-          image_name=image_name, sshd_systemd_src_file=sshd_systemd_src_file, systemd_dst_path=systemd_dst_path))
-        cmd_kvm_adj.append("virt-copy-in -a /tmp/{image_name}.snapshot.img {sshd_config_src_file} /etc/ssh/".format(
-          image_name=image_name, sshd_config_src_file=sshd_config_src_file))
-        cmd_kvm_adj.append("guestfish -a /tmp/{image_name}.snapshot.img -i chown 0 0 {sshd_config_dst_file}".format(
-          image_name=image_name, sshd_config_dst_file=sshd_config_dst_file))
-        cmd_kvm_adj.append("guestfish -a /tmp/{image_name}.snapshot.img -i chmod 0644 {sshd_config_dst_file}".format(
-          image_name=image_name, sshd_config_dst_file=sshd_config_dst_file))
-        # Create a symlink since Debian watches for type 'link'
-        cmd_kvm_adj.append(("guestfish -a /tmp/{image_name}.snapshot.img -i ln-s ".format(image_name=image_name) +
-          "{systemd_path}sshd-integration.test.service ".format(systemd_path=systemd_dst_path) +
-          "{systemd_path}multi-user.target.wants/sshd-integration.test.service".format(
-            systemd_path=systemd_dst_path)))
-
-        for i in cmd_kvm_adj:
-            logger.info("Running: {cmd}".format(cmd=i))
-            p = subprocess.run([i], shell=True, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
+
+        # Copy some files to the snapshot
+        copy_cmd = (
+            "virt-copy-in -a /tmp/{image_name}.snapshot.qcow2 "
+            "{authorized_keys_file} {sshd_systemd_file} {sshd_config_file} "
+            "/root".format(
+                image_name=image_name,
+                authorized_keys_file=authorized_keys_file,
+                sshd_systemd_file=sshd_systemd_file,
+                sshd_config_file=sshd_config_file
+            )
+        )
+        cmds.append(copy_cmd)
+
+        # Modify the snapshot via guestfish
+        guestfish_cmd = (
```

Comment:
Probably, this is a good time to get rid of this old format of `"{foo}".format(foo=bar)` and switch to `f"{bar}"`